components/CommonButton/index.js

```diff
@@ -2,32 +2,55 @@ import styled from "styled-components";
 
 export default function CommonButton({
   ButtonName,
-  disabled,
+  isDisabled,
   onClick,
   warningButton,
+  loginButton,
```

Comment:
As you don't use typescript try adding `is` or `has` prefixes to Boolean value. As I understand from the further code it's a boolean value, so better to call it `isLoginButton`

Reply:
Same for `warningButton`

Reply:
Done :)